lib/urlwatch/command.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
```

Comment:
Remove the shebang lines in all files in /lib/urlwatch/, as they are not supposed to be run directly.